.github/workflows/largemodel_unit_test_CI.yml

```diff
@@ -82,7 +82,7 @@ jobs:
 
       - name: Install dependencies
         run: |
-          pip install -r marqo-base/requirements.txt
+          pip install -r marqo-base/requirements/amd64-gpu-requirements.txt
```

Comment:
What about the api tests pipelines? Do they need to be changed as well or is the change in api-test repo?